packages/commercetools/theme/components/SfAddressPicker.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div class="sf-address-picker">
+    <!--@slot Default. Here you should pass your addresses-->
+    <slot />
+  </div>
+</template>
+<script>
+import SfAddress from '~/components/SfAddress.vue';
+
+import Vue from 'vue';
+Vue.component('SfAddress', SfAddress);
+export default {
+  name: 'SfAddressPicker',
+  props: {
+    value: {
+      type: String | Number,
+      default: ''
+    }
+  },
+  data() {
```

Comment:
compositon api please

Reply:
It just copy paste with small change from SFUI

Reply:
yup @andrzejewsky this component will be removed once sfui 0.9 will be released